data_spec_validator/spec/validators.py

```diff
@@ -370,3 +371,27 @@ def validate(value, extra, data):
         except:
             ok = False
         return ok, ValueError(f'{value} is not an UUID object')
+
+
+class RegexValidator(BaseValidator):
+    name = REGEX
+
+    @staticmethod
+    def validate(value, extra, data):
+        regex_param = extra.get(RegexValidator.name, {})
+        pattern = regex_param.get('pattern', '')
+        match_method = regex_param.get('method', 'search')
+        error_regex_param = regex_param.copy()
+        if 'method' not in regex_param:
+            error_regex_param['method'] = 'search'
```

Comment:
is `error_regex_param['method'] = match_method` better?

Reply:
Nice, addressed